src/hooking/hooking.hpp

```diff
@@ -200,6 +200,8 @@ namespace big
 		static bool update_session_advertisement(int profile_index, MatchmakingId* id, int num_slots, int available_slots, rage::rlSessionInfo* info, MatchmakingAttributes* data, rage::rlTaskStatus* status);
 		static bool unadvertise_session(int profile_index, MatchmakingId* id, rage::rlTaskStatus* status);
 		static void send_session_detail_msg(rage::netConnectionManager* mgr, rage::netConnection::InFrame* request_frame, rage::rlSessionDetailMsg* msg);
+
+		static uint32_t net_bundle_get_data_size(__int64 _this);
```

Comment:
int64_t can be used instead of __int64 i'm pretty sure

Reply:
He did that because he just pasted it from IDA.

![image](https://github.com/YimMenu/YimMenu/assets/100792176/e8f9ec23-5cb0-4c49-8d5b-547850df6979)

Because he's an actual reverse engineer, unlike you.

Reply:
Obviously, I know that ida puts it as __int64. My review highlights the use of __int64 when int64_t is used everywhere else in the codespace. As someone who fears the compiler so much, I am surprised you weren't on this before.
Here is an example from RDR2(I reversed this function based on what calls it).
![image](https://github.com/YimMenu/YimMenu/assets/80224521/9d06f4f0-a9bb-4524-9aa8-5b0273432802)